lib/ansible/modules/network/eos/_eos_template.py

```diff
@@ -33,7 +35,7 @@
     commands that are not already configured.  The config source can
     be a set of commands or a template.
 deprecated: Deprecated in 2.2. Use eos_config instead
```

Comment:
While you are editing this can you update:
`deprecated: Deprecated in 2.2. Use M(eos_config) instead`